### PR TITLE
First pass at Horizontal bars

### LINKED ISSUE #402: Bars doesn't display the correct y value for a Tooltip
The `y` value in `mark_data` is set to the difference between the element value and the base value.

### PR DESCRIPTION
**Pull request by dmadeka**
@ssunkara1 @SylvainCorlay If we merge the `HeatMap` I think we will have all relevant Marks covered I believe.

Fixes #402

### CODE REVIEW

**Comment by dmadeka** [comment]
@SylvainCorlay The issue is we dont treat `x` and `y` symmetrically. For example, most Selectors would not work on the horizontal Bars.

**Comment by dmadeka** [comment]
@ssunkara1 Need to consider two things in this

- [ ] The view padding isnt enough for Horizontal Bars
- [ ] Date Scales don't work currently
- [ ] The stack logic doesnt work when the base is not 0

**Comment by dmadeka** [comment]
Fixes #405, #402,

**Comment by dmadeka** [comment]
@ssunkara1 Ready to go

File: examples/Marks/Bars.ipynb [hunk 1]
```diff
@@ -65,16 +98,17 @@
...
    },
    "outputs": [],
    "source": [
     "x_ord = LinearScale()\n",
     "y_sc = LinearScale()\n",
     "\n",
-    "bar = Bars(x=x_data[:20], y=np.abs(y_data_2[:20]), scales={'x': x_ord, 'y': y_sc}, base=1.0)\n",
-    "ax_x = Axis(scale=x_ord)\n",
-    "ax_y = Axis(scale=y_sc, orientation='vertical', tick_format='0.2f')\n",
+    "bar = Bars(x=x_data[:20], y=np.abs(y_data_2[:20]), scales={'x': x_ord, 'y': y_sc}, base=1.0,\n",
```
Comment (ssunkara1) [inline examples/Marks/Bars.ipynb]: remove the orientation `vertical` and change the orientation of the axes?

**Comment by ssunkara1** [comment]
All looks good with this. @dmadeka Thanks!

**Pull request merged**